mos_tests/environment/ssh.py

```diff
@@ -205,17 +205,20 @@ def execute(self, command, verbose=False):
         }
         for line in stdout:
             result['stdout'].append(line)
-            if verbose:
-                logger.info(line)
         for line in stderr:
             result['stderr'].append(line)
-            if verbose:
-                logger.info(line)
         result['exit_code'] = chan.recv_exit_status()
         stdin.close()
         stdout.close()
         stderr.close()
         chan.close()
+        if verbose:
+            logger.debug('{} exit_code is {}'.format(command,
```

Comment:
please use {0} {1} instead of {}:

logger.debug('{0} exit_code is {1}'.format(command, result['exit_code']))